ios/UIViewController+RNScreens.m

```diff
@@ -34,6 +34,15 @@ - (UIStatusBarAnimation)RNSPreferredStatusBarUpdateAnimation
   return [self RNSPreferredStatusBarUpdateAnimation];
 }
 
+- (UIInterfaceOrientationMask)RNSSupportedInterfaceOrientations
```

Comment:
typically method names start with lowercase in Objc. Also, the capital leter prefix is typically used for global names (class names, enums, exported functions) that, in Objc cannot be scoped/namespaced (this is the reason why the prefixes are used). When you define a class instance method it is already scoped to that class, so no need to add the prefix

Reply:
I followed the naming method for swizzling from `expo`: https://github.com/expo/expo/blob/3356f86046b0107a228cd9dd9cebe81e5fb39321/packages/@unimodules/react-native-adapter/ios/UMReactNativeAdapter/Services/UMReactFontManager.m#L14-L35 (cc @sjchmiela)